kevm-pyk/src/kevm_pyk/cli.py

```diff
@@ -303,7 +303,7 @@ def explore_args(self) -> ArgumentParser:
             dest='failure_info',
             default=True,
             action='store_true',
-            help='Show failure summary for all failing tests',
+            help='Show failure summary for all failing tests (default)',
```

Comment:
I know this dot wasn't there, but let's unify the dots in help messages by adding it there (it may also be good to add one for `--no-failure-information`):
```suggestion
            help='Show failure summary for all failing tests (default).',
```